src/components/admin/withdrawals/EditForm.tsx

```diff
@@ -47,7 +47,7 @@ const validationSchema: yup.SchemaOf<WithdrawalData> = yup
   .defined()
   .shape({
     status: yup.string().trim().min(1).max(10).required(),
-    amount: yup.number().positive().integer().required(),
+    amount: yup.number().positive().required(),
```

Comment:
Not that it is an issue considering the amount field can't be edited in EditForm, but maybe for consistency reason we should at least have the number().positive(), rules for amount validation.